src/DIRAC/RequestManagementSystem/DB/RequestDB.py

```diff
@@ -237,8 +236,13 @@ def cancelRequest(self, requestID):
         try:
             updateRet = session.execute(
                 update(Request)
-                .where(Request.RequestID == requestID)
-                .values({Request._Status: "Canceled", Request._LastUpdate: datetime.datetime.utcnow()})
+                .where(Request.RequestID == requestID)  # pylint: disable=no-member
+                .values(
+                    {
+                        Request._Status: "Canceled",  # pylint: disable=no-member
```

Comment:
that's a lot of cosmetic changes, which makes it hard to review. 
Why did you add all this `pylint` hints ?

Reply:
Because there was a `pylint: disable=no-member` applied for the entire file (which I removed, as it is bad practice).